src/modules/0.0.14/aws_vpc/index.ts

```diff
@@ -543,8 +579,17 @@ export const AwsVpcModule: Module2 = new Module2({
               await AwsVpcModule.mappers.vpc.db.update(cloudRecord, ctx);
               out.push(cloudRecord);
             } else {
-              await AwsVpcModule.mappers.vpc.cloud.create(e, ctx);
-              out.push(e);
+              const isUpdate = Object.is(AwsVpcModule.mappers.vpc.cloud.updateOrReplace(cloudRecord, e), 'update');
+              if (!isUpdate) {
+                // if CIDR is different we do the create
+                const newVpc = await AwsVpcModule.mappers.vpc.cloud.create(e, ctx);
+                out.push(newVpc);
+              } else {
+                if (!AwsVpcModule.utils.eqTags(e.tags, cloudRecord.tags) && e.vpcId && e.tags) {
```

Comment:
I would remove this last `&& e.tags` it should still update the tags if the user has deleted all of them.

```suggestion
                if (!AwsVpcModule.utils.eqTags(e.tags, cloudRecord.tags) && e.vpcId) {
```